pkg/api/auth/basic.go

```diff
@@ -13,11 +13,11 @@ import (
 // UserPropertyName allows to query the HTTP context for the current user's name.
 // See Context() on http.Request.
 //
-//   func(w http.ResponseWriter, r *http.Request) {
-//     user := r.Context().Value(auth.UserPropertyName);
-//     fmt.Fprintf(w, "This is an authenticated request")
-//     fmt.Fprintf(w, "User name: '%s'\n", user)
-//   }
+//	func(w http.ResponseWriter, r *http.Request) {
+//	  user := r.Context().Value(auth.UserPropertyName);
+//	  fmt.Fprintf(w, "This is an authenticated request")
+//	  fmt.Fprintf(w, "User name: '%s'\n", user)
+//	}
```

Comment:
is commented code necessary?

Reply:
This are AFAIK example code snippets for go doc. So kind of?  Also I didn't touch this, that's just automatic formatting change